core/logs/src/main/scala/org/typelevel/otel4s/logs/Logger.scala

```diff
@@ -53,6 +56,58 @@ sealed trait Logger[F[_], Ctx] {
 object Logger {
   private[otel4s] trait Unsealed[F[_], Ctx] extends Logger[F, Ctx]
 
+  sealed trait Meta[F[_], Ctx] {
+
+    /** Indicates whether instrumentation is enabled.
+      *
+      * @param context
+      *   the context to be associated with the log record
+      *
+      * @param severity
+      *   the severity of the log record
+      *
+      * @param eventName
+      *   the event name to be associated with the log record
+      *
+      * @return
+      *   `true` if instrumentation is enabled, `false` otherwise
+      *
+      * @see
+      *   [[https://opentelemetry.io/docs/specs/otel/logs/api/#enabled]]
+      */
+    def isEnabled(context: Ctx, severity: Option[Severity], eventName: Option[String]): F[Boolean]
+
+    /** Modify the context `F` using the transformation `f`. */
+    def mapK[G[_]](f: F ~> G): Meta[G, Ctx] =
```

Comment:
should we actually allow transforming `Meta` instances using arbitrary natural transformations, or should we only support lifting the type? this question obviously applies to all `<Something>Meta` instances, not just `Logger.Meta`

Reply:
Good idea. I think we can start with `liftTo` only.